src/main/kotlin/com/openlattice/shuttle/Shuttle.kt

```diff
@@ -526,6 +527,7 @@ class Shuttle (
             val flightNames = flightPlan.keys.joinToString(", ") { it.name }
             val exceptionLog = "Encountered exception $ex while integrating flight plan containing flight(s) $flightNames"
             writeLog(flightNames, setOf(exceptionLog, ex.stackTrace.toString()), IntegrationStatus.FAILED)
+            if (!isShuttleServer) throw java.lang.Exception(exceptionLog)
```

Comment:
this should call `MissionControl.fail()` instead of throwing, probably

Reply:
it might also be worth adding a comment that this line is to preserve legacy integrations that haven't been updated yet, and should be removed once that method of integration is deprecated